src/screens/Game/GamePage/index.tsx

```diff
@@ -478,7 +483,7 @@ export default function GamePage(): JSX.Element | null {
         return refresh()
       }
 
-      if (installPath === 'default') {
+      if (installPath === 'default' && gameInfo.namespace != 'ue') {
```

Comment:
Same here. Maybe is better to just remove the import and install on default path from the option list.